libsolidity/analysis/DocStringTagParser.cpp

```diff
@@ -47,6 +47,14 @@ bool DocStringTagParser::parseDocStrings(SourceUnit const& _sourceUnit)
 	return errorWatcher.ok();
 }
 
+bool DocStringTagParser::validateDocStringsUsingTypes(SourceUnit const& _sourceUnit)
+{
+	m_resolverRan = true;
```

Comment:
Wouldn't it be more appropriate to call it something like `m_validateDocStringsUsingTypesRan`? The class can't really know if the resolver step actually ran before it but the current name makes you think it does.